extensions/interactions/RatioExpressionInput/directives/ratio-expression-input-validation.service.ts

```diff
@@ -79,6 +87,32 @@ export class RatioExpressionInputValidationService {
       this.baseInteractionValidationServiceInstance.getAllOutcomeWarnings(
         answerGroups, defaultOutcome, stateName));
 
+    let rulesService = this;
+
+    // Checks whether currentInput is in simplest form or not.
+    let isInSimplestForm = function(
+        currentRuleType: string,
+        ratio: Ratio,
+        currentInput: number[]
+    ): boolean {
+      return currentRuleType === 'IsEquivalent' && (
+        !rulesService.rof.arrayEquals(
+          ratio.convertToSimplestForm(), currentInput));
+    };
+
+    // Checks whether currentInput has less number of terms than seenInput.
+    let hasLessNumberOfTerms = function(
+        currentRuleType: string,
+        seenRuleType: string,
+        currentInput: number,
+        seenInput: number[]
+    ): boolean {
+      return currentRuleType === 'HasNumberOfTermsEqualTo' &&
+      seenRuleType !== 'HasNumberOfTermsEqualTo' && (
```

Comment:
Suggest adding parenthesis to the return, e.g. ``return (currentRuleType...)`` then fix the indentation here to be correctly aligned & continued (we always continue & align at 2 spaces further than the indentation of the line containing the open parenthesis.

Reply:
done

Reply:
Doesn't seem done--``seenRuleType...`` is still not indented as expected.

Reply:
Suggest adding a ``(`` after the ``&&`` to fix indentation. I actually think the linter is wrong here.

@bansalnitish & @Hudda FYI--is the linter correctly prohibiting indentation here? It doesn't seem correct since nested expressions should continue indentation from their parent nodes in the AST.

Reply:
FYI, the usual convention for this sort of thing is:

```
      return (
        currentRuleType === 'HasNumberOfTermsEqualTo' &&
        seenRuleType !== 'HasNumberOfTermsEqualTo' &&
        ratioRulesService.HasNumberOfTermsEqualTo(
          seenInput, {y: currentInput}));
```

I.e. if a statement spans more than 1 line overall, add a "(" at the beginning of it and break after the "(".